app/controllers/people_controller.rb

```diff
@@ -16,7 +16,10 @@ def show
         response.headers['Content-Length'] = @person.v_card.size.to_s
       }
       format.pdf {
-          pdf = Services::Token.generate_token_pdf(@person, settings.token_media)
+          Rails.logger.info "settings #{settings.token_media}"
+          
+          pdf = Services::Token.generate_token_pdf(@person, settings.token_media,
+                                        ServiceProvider.find_by_special_role(:registration).token_layout)
```

Comment:
we could go crazy and allow to generate tokens for each service provider, I am not sure, but maybe we should get rid of an service provider agnostic view.
woudnt it be cool if we could generate a pdf somehow like this: /service_providers/registration/users/ID123.pdf